BLSQ/iaso
# IA-2629, IA-2631:  make doc edition easier

it was impossible to preview images when editing docs on github

Related JIRA tickets : IA-2629, IA-2631

## Self proofreading checklist


## Changes

- Update yaml config: remove unneeded plugin, add markdown extension allowing setting image style
- Replace `img` tags in web user guide with markdown links

## How to test

The branch is currently deployed at https://iaso.readthedocs.io/en/ia-2617_make_doc_edition_easier/pages/users/reference/user_guide/ (so not as the latest build).

All images in the user guide should till appear, with the same size as the main doc page: https://iaso.readthedocs.io/en/latest/pages/users/reference/user_guide/user_guide/

Go the [branch's user guide](https://github.com/BLSQ/iaso/blob/IA-2617_make_doc_edition_easier/docs/pages/users/reference/user_guide.md): the images should display. You should be able to edit height and width.

Doc to add a new image will come as another PR



## Print screen / video

![Screenshot 2024-01-12 at 19 23 42](https://github.com/BLSQ/iaso/assets/38907762/b50c2621-fd06-4e1a-81c2-cc2146fd993d)

![Screenshot 2024-01-12 at 19 24 04](https://github.com/BLSQ/iaso/assets/38907762/d91b1f5a-4b8e-4ff7-b8f2-d5c5e7831a36)
